components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/Config.kt

```diff
@@ -9,23 +9,28 @@ package mozilla.appservices.fxaclient
  * authentication flow.
  */
 class Config constructor(
-    val contentUrl: String,
+    val server: FxaServer,
     val clientId: String,
     val redirectUri: String,
     val tokenServerUrlOverride: String? = null,
 ) {
-    enum class Server(val contentUrl: String) {
-        RELEASE("https://accounts.firefox.com"),
-        STABLE("https://stable.dev.lcip.org"),
-        STAGE("https://accounts.stage.mozaws.net"),
-        CHINA("https://accounts.firefox.com.cn"),
-        LOCALDEV("http://127.0.0.1:3030"),
+    enum class Server(val rustServer: FxaServer) {
```

Comment:
One interesting issue with this refactor is that it seems impossible for consuming apps to discover what the actual URL being used it? ie, they might construct with `FxaServer.Stage`, but later when they want to open, say, the "manage account" page, they seem unable to discover what the base URL is. Before this change they are able to obtain it via the enum.

Not sure what the answer is here - ie, whether we should expose the base URL, or expose a way to build a URL - eg, how desktop does it is [here](https://searchfox.org/mozilla-central/source/services/fxaccounts/FxAccountsConfig.sys.mjs#79) - the code is responsible for knowing whether the email address should be included in the URL etc.